lit_gpt/config.py

```diff
@@ -978,6 +1018,26 @@ def norm_class(self) -> Type:
         intermediate_size=22016,
         rope_base=1000000,
     ),
+    # https://huggingface.co/codellama/CodeLlama-70b-Instruct-hf/blob/main/config.json
+    dict(
+        name="CodeLlama-70b-Instruct-hf",
+        hf_config=dict(org="codellama", name="CodeLlama-70b-Instruct-hf"),
+        block_size=16384,
+        vocab_size=32016,
+        padding_multiple=16,
```

Comment:
@carmocca would you recommend using padded_size here as well?

Reply:
Doesn't matter, vocab_size+padding_multiple=padded_size